app/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <declare-styleable name="GlobalAttributeDeclare">
+        <attr name="icon" format="reference"/>
+        <attr name="title" format="string"/>
+        <attr name="android:inputType"/>
+    </declare-styleable>
+
+    <declare-styleable name="DetailInformationCustomView">
+        <attr name="icon"/>
+        <attr name="title" />
+    </declare-styleable>
+
+    <declare-styleable name="AddPropertyInformationCustomView">
+        <attr name="icon"/>
+        <attr name="title"/>
```

Comment:
If it doesn't work then you can turning it into a public field you can get and set outside of the CustomView.

Reply:
work fine, i have forgot to bind attribute in the custom view code.